src/fake.ts

```diff
@@ -88,16 +88,26 @@ export class Fake {
     // split the method into module and function
     const parts = method.split('.');
 
-    if (this.faker[parts[0]] == null) {
-      throw new FakerError(`Invalid module: ${parts[0]}`);
+    let currentModuleOrMethod: unknown = this.faker;
+    let currentDefinitions: unknown = this.faker.definitions;
+
+    for (const part of parts) {
+      currentModuleOrMethod = currentModuleOrMethod?.[part];
+      currentDefinitions = currentDefinitions?.[part];
     }
 
-    if (this.faker[parts[0]][parts[1]] == null) {
-      throw new FakerError(`Invalid method: ${parts[0]}.${parts[1]}`);
+    let fn: (args?: unknown) => string;
```

Comment:
In theory, this could return anything not only strings. So typing the return type as unknown to catch unhandled cases should be more secure IMO.

```suggestion
    let fn: (args?: unknown) => unknown;
```